### ISSUE TITLE
`.env.[target-configuration-name]` is loaded by not documented

### ISSUE THREAD

**Issue by wei-fh** [nrwl/nx#27612]
### Documentation issue

- [ ] Reporting a typo
- [x] Reporting a documentation bug
- [ ] Documentation improvement
- [ ] Documentation feedback

### Is there a specific documentation page you are reporting?

https://nx.dev/recipes/tips-n-tricks/define-environment-variables

### Additional context or description

The .env* file patterns listed on this page don't include `.env.[target-configuration-name]` (without the [target-name]), but by my testing it's indeed loaded.

nx version v19.5.6

**Comment by FrozenPandaz** [comment]
Yes, it is loaded. Would you like to contribute to the documentation please?

**Comment by wei-fh** [comment]
Thanks for the confirmation. I'd love to update the document but it's hard for me to identify the exact priority of the loading.
